### ISSUE TITLE
Elytra issue. 

### ISSUE THREAD

**Issue by CLGREEN94** [andune/HomeSpawnPlus#106]
Hello,  just wondering if you are aware of the issue that if you have a Elytra on your back and do /Spawn it will pop it in your inventry but if you have no space in the inventry it will just remove it. 

I have test to see if it was HSP and found it was I used Multiverse and Tp'ed to another world and it did not remove the elytra. 

I cant see to access the Jenkins server to see if you have patched the files. 

Thanks, 
Callum.

**Comment by sorifiend** [comment]
From what I can tell: HSP is not coded in such a way that it messes with your inventory, I suspect another plugin is causing your problem to occur (most likely your inventory plugin or multi world plugin).

Having said that, I did give this a test on a 1.10 server using the latest build of HSP (2.0.1-SNAPSHOT-b637), and I could not seem to reproduce the issue.

If you haven't already, then try it again on a fresh 1.10 spigot server with only the HSP plugin installed, then add plugins one by one until it starts happening again, then you will find out what plugin is interfering with your items.

**Comment by SimonOrJ** [comment]
My server has no elytra issues as well. It might be another plugin.

**Comment by sorifiend** [comment]
Just an update on this: Jenkins is back online, so give the latest version a shot if you haven't already.

**Comment by CLGREEN94** [comment]
Hey guys, 

Thanks for all the help, sorry for only just getting back to you. We did find the issue it was 'Races and Classes'. For some odd reason when using Multiverse to TP to a different world the issue did not occur but when using HSP it did.

We can only assume this is down to HSP must tell 'Races and Classes' the payer has actually changed the world and this ten tells 'Races and Classes' to check to see if the players are allowed the said item in there armour slot. 

As we did not have it set in the config to allow the item then it would try and remove the item from the slot and pop it in the players inventory but if the inventory was full it would just remove the item all together, something I need to bring up with the Races and Classes Dev.  Multiverse must not tell 'Races and Classes' that the player has changed world if that is a good or bad thing I don't know. 

Thanks for your time :+1:
